tools/wptserve/wptserve/pipes.py

```diff
@@ -484,6 +488,8 @@ def config_replacement(match):
             value = request.server.config.all_domains[""]
         elif field == "host":
             value = request.server.config["browser_host"]
+        elif field == "ip_address":
+            value = request.server.config["server_host"]
```

Comment:
Question: do we guarantee that `server_host` is an IP? Can it be e.g. `localhost`? Even if the default is an IP, some people might overwrite it to `localhost` in their custom config without noticing (as everything else works regardless I assume).

Reply:
?